Source/Tests/LinuxSyscalls/x32/Syscalls.cpp

```diff
@@ -45,17 +45,17 @@ namespace FEX::HLE::x32 {
   void RegisterTime();
   void RegisterTimer();
 
-  std::map<int, const char*> SyscallNames = {
-    #include "SyscallsNames.inl"
-  };
-
-  const char* GetSyscallName(int SyscallNumber) {
-    const char* name = "[unknown syscall]";
+  [[nodiscard]] static const char* GetSyscallName(int SyscallNumber) {
```

Comment:
Would it also make sense to to wrap these `GetSyscallName` functions with the same define check as the usage `#if  defined(ASSERTIONS_ENABLED) && ASSERTIONS_ENABLED` to completely remove it from release builds?

Reply:
Ah true. I've updated the commit so that `GetSyscallName()` is wrapped in the ifdef check